.github/ISSUE_TEMPLATE/Mobile_apps_bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report for Mobile Apps
+about: Create a report to help us improve the Gutenberg mobile apps version
+labels: Mobile App Android/iOS
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Tap on '....'
+3. Scroll down to '....'
+4. See '... exact error ...'
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Editor version (please complete the following information):**
+- WordPress version: [e.g: 5.3.2]
+- Does the website has the Gutenberg plugin installed, or is it using the block editor that comes by default? [e.g: "gutenberg plugin", "default"]
+- If the Gutenberg plugin is installed, which version is it? [e.g., 7.6]
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6, Pixel 3]
+ - OS: [e.g. iOS 8.1, Android 18.1]

```

Comment:
Minor nit, but let's change the Android version to something like 10.0. Having 18.1 here could be a bit confusing since 10.0 is the latest version and the API levels are approaching 30, but they don't have point numbers.